ui/raidboss/data/05-shb/dungeon/malikahs_well.js

```diff
@@ -189,5 +190,36 @@
         'Armadillo': 'Armadillo',
       },
     },
+    {
+      'locale': 'ko',
+      'replaceSync': {
+        '00:0839:Terminus will be sealed off': 'Drehscheibe',
```

Comment:
Did you mean for these to not have a translation into Korean?

Reply:
I missed that. thanks.